src/app/shared/pipes/app-panel-show-details-aria-label/app-panel-show-details-aria-label.pipe.ts

```diff
@@ -0,0 +1,82 @@
+import { Pipe, PipeTransform } from '@angular/core'
+import { ModalEmailComponent } from '../../../cdk/side-bar/modals/modal-email/modal-email.component'
+import { ModalCountryComponent } from '../../../cdk/side-bar/modals/modal-country/modal-country.component'
+import { ModalWebsitesComponent } from 'src/app/cdk/side-bar/modals/modal-websites/modal-websites.component'
+import { ModalKeywordComponent } from 'src/app/cdk/side-bar/modals/modal-keyword/modal-keyword.component'
+import { ModalPersonIdentifiersComponent } from 'src/app/cdk/side-bar/modals/modal-person-identifiers/modal-person-identifiers.component'
+import { ModalNameComponent } from 'src/app/record/components/top-bar/modals/modal-name/modal-name.component'
+import { ModalBiographyComponent } from 'src/app/record/components/top-bar/modals/modal-biography/modal-biography.component'
+import {
+  PANEL_TYPE_PLACEHOLDER,
+  PANEL_TITLE_PLACEHOLDER,
+} from 'src/app/constants'
+
+@Pipe({
+  name: 'appPanelShowDetailsAriaLabel',
+})
+export class AppPanelShowDetailsAriaLabelPipe implements PipeTransform {
```

Comment:
Instead of having 4 pipes with almost the same logic, would it be a good idea to past `delete` `edit` `hide` or `show` as another pipe parameter?

Reply:
Refactored, now we he one pipe for the item actions edit, show, hide, delete